File: 2485-find-the-pivot-integer/2485-find-the-pivot-integer.java
import java.lang.Math;
class Solution {
    public int pivotInteger(int n) {
    if(n==1) return 1;
    if(n<8) return -1;
    double x;
    x=n*n;
    x+=n;
    x/=2;
    x=Math.sqrt(x);
    if (x%1==0) return (int)x;
    else return -1;
    

    }
}

File: 2485-find-the-pivot-integer/2485-find-the-pivot-integer.c
#include<math.h>
int pivotInteger(int n) {
    if(n==1) return 1;
    if(n<8) return -1;
    double x;
    x=n*n;
    x+=n;
    x/=2;
    x=sqrt(x);
    if (floor(x) == x)
        return (int)x;
    else return -1;
    
}